demo-shell/src/app/app.component.html

```diff
@@ -1,2 +1,4 @@
-<router-outlet></router-outlet>
-<router-outlet name="overlay"></router-outlet>
+<div [dir]="textOrientation" class="adf-demo-app-container" >
```

Comment:
we may need to document that client apps may want to have this set around router outlet

Reply:
https://github.com/Alfresco/alfresco-ng2-components/blob/ae12e3f01978a50225661e3ad517f53f2d1f8737/docs/user-guide/rtl-support.md already done